src/math/libm.cr

```diff
@@ -1,4 +1,8 @@
-{% if flag?(:linux) || flag?(:bsd) %}
+# Interpreter: On GNU systems, libm.so is typically a GNU ld script which adds
+# the actual library file to the load path. `Crystal::Loader` does not support
+# ld scripts yet. So we just skip that for now. The libm symbols are still
+# available in the interpreter.
+{% if (flag?(:linux) && !flag?(:interpreted)) || flag?(:bsd) %}
```

Comment:
You'd want to check the interpreter flag regardless I guess.

```suggestion
{% if (flag?(:linux) || flag?(:bsd)) && !flag?(:interpreted) %}
```

Reply:
No, it's actually specific to GNU/Linux. At least that's my current level of knowledge.
It could even be more restricted with `flag?(:gnu)`, but that's going to happen in #11563 anyways.